python/ray/rllib/a3c/runner.py

```diff
@@ -29,19 +31,16 @@ def __init__(self, env_creator, config, logdir):
         obs_filter = get_filter(
             config["observation_filter"], env.observation_space.shape)
         self.rew_filter = get_filter(config["reward_filter"], ())
-
-        self.sampler = AsyncSampler(env, self.policy, config["batch_size"],
-                                    obs_filter)
+        self.sampler = AsyncSampler(env, self.policy, obs_filter,
+                                    config["batch_size"])
         self.logdir = logdir
 
-    def get_data(self):
+    def sample(self):
         """
         Returns:
-            trajectory: trajectory information
-            obs_filter: Current state of observation filter
-            rew_filter: Current state of reward filter"""
-        rollout, obs_filter = self.sampler.get_data()
-        return rollout, obs_filter, self.rew_filter
+            trajectory (PartialRollout): Experience Samples from evaluator"""
+        rollout = self.sampler.get_data()
+        return rollout
```

Comment:
Hm, in order to support concatenation, the returned object should be a SampleBatch. This is required if you want to swap in the multi-gpu optimizer or synchronous one (the async one never concats samples).

This doesn't have to be addressed in this pr.